src/DynamoMLDataPipeline/DynamoMLDataPipeline.cs

```diff
@@ -184,7 +197,7 @@ static void DataExchangeToForge(string filePath, RestClient client, string token
             // from the response - these will be consumed by the following API calls.
             ExchangeContainerId = exchangeRequestResponseBody["id"].Value;
 
-            Analytics.TrackEvent(Actions.Export, Categories.DynamoMLDataPipelineOperations, "ExchangeContainerID", Convert.ToInt32(ExchangeContainerId));
+            Analytics.TrackEvent(Actions.Export, Categories.DynamoMLDataPipelineOperations, ExchangeContainerId);
```

Comment:
Is there a reason we send the data (CollectionId, ExchangeContainerId, BinarySize) as different Analytics events? Would it be possible to combine them into one Analytics log given that they are all part of the same export event? Analytics.NET seems to have an Analytics.TrackEvent call that accepts a dictionary of values. 
Ideally, on the ADP side we would like to be able to map CollectionId to ExchangeContainerId or monitor the BinarySize for a given collection.

Reply:
Yes changed it to a dictionary.